### PR TITLE
Addition of Protective Marking sub-section to api-security.md

### PR DESCRIPTION
**Pull request by TimGoodwill**
Addition of Protective Marking sub-section to API Security, as discussed Working Group July 27. 
A unified approach to protective marking would greatly enhance API inter-operabilty. It is acknowledged that each jurisdiction may have similar requirements, and this section should be open to evolution.
I have based this addition on Peeyush's PR update to this document.

### CODE REVIEW

**Review by jordanwalsh23 — CHANGES_REQUESTED** [review]
I'm happy with these changes from a Victorian Government API standards viewpoint.  We are comfortable to implement the protective marking approach with the APIs we build in the WoVG API Team that contain sensitive data.

That said, the bullet points in the 2nd paragraph have slightly gone out of alignment, can you resolve and resubmit?

**Comment by TimGoodwill** [comment]
Updated as requested

File: sections/api-security.md [hunk 1]
```diff
@@ -31,12 +31,12 @@ Depending on the security classification you may be required to establish the fo
 ## Authentication and Authorization
 
 - Basic or Digest authentication **SHOULD NOT**  be used.
-- The `Authorization: Bearer` header **MUST** be used for authentication/authorization e.g. using a JWT token.
+- The `Authorization: Bearer` header **MUST** be used for authentication/authorization e.g. using a JWT token. Authorization tokens identify a user — the person — that is using the app or site.
 - A refresh token **SHOULD** be provided for extending expiry time of existing token without having to provide the credentials again. 
 - Always set a reasonable expiration date for tokens. JWT token lifetime **SHOULD NOT*** exceed 5 minutes.
 - JWT refresh tokens **SHOULD** be used when new JWT tokens are required outside of this lifetime.
```
Comment (ghost) [inline sections/api-security.md:37]: Refresh tokens do not have to be JWT, most are opaque

File: sections/api-security.md [hunk 2]
```diff
@@ -31,12 +31,12 @@ Depending on the security classification you may be required to establish the fo
 ## Authentication and Authorization
 
 - Basic or Digest authentication **SHOULD NOT**  be used.
-- The `Authorization: Bearer` header **MUST** be used for authentication/authorization e.g. using a JWT token.
+- The `Authorization: Bearer` header **MUST** be used for authentication/authorization e.g. using a JWT token. Authorization tokens identify a user — the person — that is using the app or site.
```
Comment (ghost) [inline sections/api-security.md]: Not every token needs to represent a person, e.g. Client credential flow.
No need for the additional text

Comment (TimGoodwill) [reply sections/api-security.md]: I would agree. Additional text removed.

File: sections/api-security.md (hunk 1 above)
Comment (TimGoodwill) [reply sections/api-security.md:37]: Updated.

**Comment by TimGoodwill** [comment]
To be replaced with generalised version, based on current v1.1

**Pull request closed without merging**